CRM/Contribute/Form/UpdateSubscription.php

```diff
@@ -209,7 +209,8 @@ public function postProcess() {
     $params['id'] = $this->getContributionRecurID();
     $message = '';
 
-    $params['subscriptionId'] = $this->getSubscriptionDetails()->processor_id;
+    $params['recurProcessorID'] = $params['subscriptionId'] = $this->getSubscriptionDetails()->processor_id;
```

Comment:
This adds `recurProcessorID` which matches what is defined as the "correct" key in PropertyBag. It's more accurate than "subscriptionId" which doesn't exist in the database by that name and good to define it now so that payment processors can choose to use that key.